protocol/daemons/server/types/util.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import "time"
+
+// MaximumAcceptableUpdateDelay computes the maximum acceptable update delay for a daemon service as a
+// multiple of the loop delay.
+func MaximumAcceptableUpdateDelay(loopDelayMs uint32) time.Duration {
```

Comment:
Small suggestion: wdyt about adding unit tests for this function? Even one should suffice